infrastructure/validators/operational-tasks/chill-your-node.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Chill Your Node
+description: Instructions on chilling as a network participant.
+---
+
+Staking bonds can be in any of the three states: validating, nominating, or chilled (neither
+validating nor nominating). When a staker wants to temporarily pause their active engagement in
+staking but does not want to unbond their funds, they can choose to "chill" their involvement and
```

Comment:
```suggestion
staking but doesn't want to unbond their funds, they can choose to "chill" their involvement and
```
We do prefer the use of contractions for 'not' combinations. Readers are more likely to see the 'not' indicator if it is written "don't" or "won't" rather than as two words they might miss one if skimming, etc.